linear_regression.q

```diff
@@ -0,0 +1,6 @@
+// Alpha and beta functions

```

Comment:
Explain that these formulas come from OLS (Ordinary Least Squares) for one variable.

Reply:
okey mackey